packages/noports_core/lib/src/sshrvd/sshrvd_params.dart

```diff
@@ -43,7 +44,7 @@ class SshrvdParams {
       managerAtsign: r['manager'],
       ipAddress: r['ip'],
       verbose: r['verbose'],
-      snoop: r['snoop'],
+      snoop: BuildEnv.enableSnoop && r['snoop'],
```

Comment:
Block the boolean condition from trying to call r['snoop'] (which will be null since we didn't add the parser flag (see the other comment in this file)